examples/command/command.cpp

```diff
@@ -578,6 +584,9 @@ int process_general_transcription(struct whisper_context * ctx, audio_async &aud
                     // prepend the prompt audio
                     pcmf32_cur.insert(pcmf32_cur.begin(), pcmf32_prompt.begin(), pcmf32_prompt.end());
 
+                    // append 1 second of silence
+                    pcmf32_cur.insert(pcmf32_cur.end(), 1000*WHISPER_SAMPLE_RATE/1000, 0.0f);
+
```

Comment:
This is needed to avoid internal whisper logic to ignore last 1 second of audio. Not related to grammar stuff, but makes the example more robust